modules/end-user-guide/pages/optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc

```diff
@@ -0,0 +1,111 @@
+:_content-type: CONCEPT
+:description: Optional parameters for the URLs for starting a new workspace
+:keywords: url, urls, parameter, parameters, starting, start, workspace, workspaces, url-parameter, url-parameters, multiple-url-parameters, combine-url-parameters, several-url-parameters, add-url-parameters, concatenate, concatenate-multiple-URL-parameters, concatenate-URL-parameters, concatenating, concatenating-multiple-URL-parameters, concatenating-URL-parameters, concatenation, start-duplicate-workspaces, start-duplicated-workspaces, duplicate-workspace, duplicate-workspaces, duplicated-workspace, duplicated-workspaces, how-to-start-a-duplicate, how-to-start-duplicate, devfile, override-devfile-file-name, override-file-name, file-name, devfile-name, different-devfile-name, file-path, devfile-path, devfile-file-path, devfile-path-parameter, devfile-file-path-parameterdevfile, file-path, devfile-path, devfile-file-path, devfile-path-parameter, devfile-file-path-parameter
+:navtitle: Optional parameters for the URLs for starting a new workspace
+// :page-aliases:
+
+[id="parameters-for-the-urls-for-starting-a-new-workspace_{context}"]
+= Optional parameters for the URLs for starting a new workspace
+
+For the URLs for starting a new workspace, the following optional parameters provide an extra layer of basic workspace configuration in addition to the devfile.
```

Comment:
This is good traditional tech writing. However, our updated style guidelines ask us to rephrase things in terms of users and their goals. For example, in that case, you could rephrase this as:
```suggestion
When you start a workspace, {prod-short} relies on the devfile for its configuration settings. If you use a URL to start a new workspace, you can append optional parameters to the URL that further configure the workspace. You can use these  parameters to specify a workspace IDE, start duplicate workspaces, and specify a devfile file name or path.
```
If users always use a URL to start a workspace, you could cut "If you use a URL to start a new workspace," from the above suggestion.

Reply:
I think the following sentence lets the reader quickly grasp whether the topic helps them, especially if it shows up in the search results. "You can use these  parameters to specify a workspace IDE, start duplicate workspaces, and specify a devfile file name or path."

Reply:
Thanks, Rolfe. I used your suggestion and made some adjustments to it:
```suggestion
When you start a workspace, {prod-short} configures the workspace according to the instructions in the devfile. When you use a URL to start a new workspace, you can append optional parameters to the URL that further configure the workspace. You can use these parameters to specify a workspace IDE, start duplicate workspaces, and specify a devfile file name or path.
```